index.html

```diff
@@ -39,19 +53,30 @@ <h3>Чем могу быть полезен</h3>
 
     <section class="portfolio">
       <div class="container">
+        <figure class="project-image">
```

Comment:
project image should be placed **under** subheading

Reply:
fixed